### PR TITLE
Hello world

### PR DESCRIPTION
**Pull request by usorfaitheloho**
## Hello microverse

In this milestone, I did the following

- create index.html 
- create style.css
- create test.md

### CODE REVIEW

File: test.md [hunk 1]
```diff
@@ -0,0 +1 @@
+"This file should be ignored by git".
```
Comment (medaminedev66) [inline test.md:1]: To maintain our git history clean, let's make sure to add `test.md` file to the [.gitignore](https://stackoverflow.com/questions/13675216/why-should-i-use-gitignore) file. What do you think? The purpose of gitignore files is to ensure that certain files not tracked by git remain untracked.

File: README.md [hunk 2]
```diff
@@ -1 +1,57 @@
...
+- HTML
+
+- CSS
+
+- Git and Github
+
+## Live Demo 
+- [Live Demo Link]
+https://livedemo.com/
+
```
Comment (medaminedev66) [inline README.md:22]: This live demo link is broken. Kindly fix this or remove it. Your README is the entry point of your users and having it well written without any broken links gives a good first impression and maximizes the potential of your product to your users.

**Review by medaminedev66 — COMMENTED** [review]
Hi @usorfaitheloho ,  :wave:

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight
- ✔️The repo looks professional
- ✔️Git/GitHub flow best practices have been followed
- ✔️There are no linter errors
- ✔️You have a descriptive README File

## Required Changes ♻️
- [ ] Kindly write a descriptive pull request title. The Pull Request should have a descriptive title telling others what changes are inside.
_See the inline comments below for more required changes_

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/V

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by HENRYKC24 — APPROVED** [review]
# STATUS: APPROVED :heavy_check_mark: 
Good Job
Hi @usorfaitheloho :wave:

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**